chromadb/api/types.py

```diff
@@ -15,7 +15,14 @@
 )
 
 # Re-export types from chromadb.types
-__all__ = ["Metadata", "Where", "WhereDocument", "UpdateCollectionMetadata"]
+__all__ = [
+    "Metadata",
+    "Where",
+    "WhereDocument",
+    "UpdateCollectionMetadata",
+    "WhereOperator",
+    "LogicalOperator",
```

Comment:
to silence some mypy errors in strategies.py (e.g `types.WhereOperator is not defined`)